Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md

```diff
@@ -3202,91 +3199,125 @@ TODO add long description + metrics, costs, personnel, related levels
 
 ##### ASSESSMENT
 * Is your penetration test continuously developed further?
 * Do you assure, that penetration tests are executed for every released hardware/software combination?
 * Does your company collect and archive all results of these penetration tests?
 
 ##### RESULTS
-* Detailed matrix which links attack patterns to ECUs
-* Database with references from software/hardware variants of ECU and possible attack patterns
-* Detailed TODO list / bug list for each product/ECU
+* Detailed matrix which links attack patterns to devices
+* Database with references from software/hardware variants of devices and possible attack patterns
+* Detailed TODO list / bug list for each product/device
 
 ##### SUCCESS METRICS
-* XXX
+* independent penentration test development team with at least one person fulltime
+* company development process contains process steps for penetration tests at least for every release
+* archiving of penetration test results is guaranteed
+* Database contains at least \>70% of released/sold products and their configuration management
 
 ##### COSTS
-* XXX
+* Ongoing project overhead from need of implementing specific security tests
+* Ongoing project overhead from need of executing security tests
+* Ongoing organization overhead from testing all relevant devices and variants
+* Ongoing organization overhead from maintaining attack pattern cataloge
+
 
 ##### PERSONNEL
-* XXX
+* Developers
+* Testers
+* Managers
+* Security Auditor
 
 ##### RELATED LEVELS
-* XXX
+* None 
 
 # Lifecycle Security Monitoring: LSM3
 ### Minimize negative impacts for customers
 
 ## Activities
-##### A. Deploy degradations to customer
-Already when starting a new ECU project, one has to clarify the communication to the end user/customer. This is essential to ensure later updates and or degradations. Therefore, an infrastructure has to be set up. This infrastructure has to be cleared with OEM also. It has to be maintained and ensured that it is "future save". Thus, it has to guarantee later updated can be deployed.
-Secondly the infrastructure is itself a high-risk channel to the ECUs in the field.
-This leads also to many efforts to be made to keep this channel secure and still working.
-Security and reliability have to be in focus.
-After in step 2 (Doing) a security update has been developed or a potential degradation is decided this has to be deployed to the ECUs out in the field. Thereby only affected ECUs should get an update. This can be dependent to Hardware and or Software version and variant. Therefore, usually at OEM side a database with software and hardware Variants have to be maintained and used for the deployment of security updates.
+##### A. Operation of deployment infrastrucure
```

Comment:
typo fix: infrastructure

Reply:
should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08